main.py

```diff
@@ -0,0 +1,41 @@
+import random
+from art import logo
+
+# level_of_difficulty() returns number of attempts based on difficulty selection
+def level_of_difficulty(choice):
```

Comment:
Some comment about type-casting with blackjack functions.  Let's do it here too.